pkg/webhook/scheduler/mutating/schedule_pod_handler.go

```diff
@@ -244,3 +239,21 @@ func (a *CreateUpdatePodForSchedulingHandler) checkIfDatasetPVCs(pvcNames []stri
 	}
 	return
 }
+
+func isFluidPod(pod *corev1.Pod) bool {
```

Comment:
Maybe renaming it to `isManagedByFluid`?

Reply:
Thanks for you suggestions! done.